.ci_support/linux_64_python3.10.____cpython.yaml

```diff
@@ -22,8 +20,6 @@ pin_run_as_build:
     max_pin: x.x
 python:
 - 3.10.* *_cpython
-pytorch:
-- '2.6'
```

Comment:
See here

Reply:
By that I mean that there's no host-dependency + run-export anymore. There's still a run-time requirement
https://github.com/conda-forge/torchdata-feedstock/blob/4c86e995017c769673304823bcf50f4c55be2e72/recipe/meta.yaml#L33-L37

so existing builds should be installable with 2.7 already

Reply:
Ahh ok that makes sense. My local env was failing and I jumped to this assumption, but armed with this knowledge I went back and discovered a pin we can lose, apologies.